hardware/RF-Test-Guide.md

```diff
@@ -0,0 +1,283 @@
+- Connect customer board and debug adapter(BRD4001A mother board) to Simplicity Studio platform.
+- Select proper radio SoCs or module and related reference board 
+- Set debug mode to "Debug Out"
+- Select new project 
+- Select RAIL: RAILTEST 
+- Define a Railtest project name 
+- Select Finish
+- Configure the radio parameters 
+  - Modulation Type: FSK2
+  - Shaping Filter: Gaussian
+  - Shaping Filter Parameter: 0.5
+  - Bitrate [bps] / Deviation [Hz]: 125k/?, 500k/125k , 1M/250k, 2M/500k
+  - Modulation index: 0.5 based on BLE standard
+  - Checked the "Reconfigure for BER testing” box
+  - Everything else is default
+- Generate 
+- Build
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\BER1.png">  
+</div>
+
+<div align="center">
+ <img src="files\HW-RF-Test-Guide\BER2.png">  
+</div>
+
+After build the Railtest tool, you can use the commander tool to program customer device, then the device will be ready for RF test.
+More information for Simplicity Studio usage, please visit the application note [AN0822](https://www.Silabs.com/documents/public/application-notes/AN0822-simplicity-studio-user-guide.pdf)
+
+## 3.3. Tune and measure
+The following strategy is proposed to determine the optimal Ctune value:
+- Performe a conducted measurement;
+- Connect the custom board's RF port to a Spectrum Analyzer(SA);
+- Use SA to measure the fundamental frequency as following picture shows:
+  <div align="center">
+  <img src="files\HW-RF-Test-Guide\CTune-CW.png">  
+  </div>
+- Make sure to set appropriate span and Resolution Bandwitdth (RBW) on the spectrum analyzer. Use a few MHz frequency range as span and a few kHz as RBW. 
+- Tune the CTune value until the frequency of the fundamental harmonic reach the optimal value. 
+- Use RailTest tool to tune CTune. The RailTest commands are listed below: 
+  - :>rx 0  
+  - :>SetPower [decidBm]  
+  - :>setdebugmode 1  
+  - :>freqoverride 868000000  
+  - :>GetCTune  
+  - :>SetCTune 0x[hex-value] or [desimal value] 
+  - :>SetTxtone 1  
+  - :>SetTxtone 0
+
+Railtest commands explaination:
+- Railtest starts with "rx 0" command. 
+- The actual output power can be set by "SetPower" command.
+- The actual output frequency can be set by "setdebugmode 1" and "freqoverride"  commands.
+- The actual CTune variable can be read by "GetCtune" command. 
+- It can set a new value to CTune, which can be done by "SetCTune" command. 
+- After the configuration, the "SetTxTone 1" command sets the radio to transmit mode. In this case the radio transmits CW signal. 
+- The CW transmission can be disabled by "SetTXtone 0" command. 
+
+For more Railtest commands information, please refer to this [KBA](https://www.Silabs.com/community/wireless/proprietary/forum.topic.html/railtest_-_more_info-66AE).
+
+For more CTune calibration details, it is discussed in this [KBA.](https://www.Silabs.com/community/wireless/proprietary/knowledge-base.entry.html/2019/03/18/hfxo_capacitor_bank-7uRt)
+
```

Comment:
Do we need to also reference programming CTUNE tokens?

Reply:
Programming CTUNE tokens is more related to software. This is just to show the calibration method.